<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Registration Test"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="14.417">
  <properties />
            <testcase name="verifySuccessfullRegistrationPageMessage" time="5.494" classname="com.demo.nopcommerce.testsuit.RegistrationTest">
                <failure type="java.lang.AssertionError"
                  message="expected [Your registration completed] but found [Your registration completed in our store]"
              >
<![CDATA[java.lang.AssertionError: expected [Your registration completed] but found [Your registration completed in our store]
  org.testng.Assert.fail(Assert.java:96)
  org.testng.Assert.failNotEquals(Assert.java:776)
  org.testng.Assert.assertEqualsImpl(Assert.java:137)
  org.testng.Assert.assertEquals(Assert.java:118)
  org.testng.Assert.assertEquals(Assert.java:453)
  org.testng.Assert.assertEquals(Assert.java:463)
  com.demo.nopcommerce.testsuit.RegistrationTest.verifySuccessfullRegistrationPageMessage(RegistrationTest.java:88)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
  org.testng.TestRunner.privateRun(TestRunner.java:648)
  org.testng.TestRunner.run(TestRunner.java:505)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
  org.testng.SuiteRunner.run(SuiteRunner.java:364)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
  org.testng.TestNG.runSuites(TestNG.java:1049)
  org.testng.TestNG.run(TestNG.java:1017)
  com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
  com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="userShouldNavigateToRegistrationPage" time="1.891" classname="com.demo.nopcommerce.testsuit.RegistrationTest" />
                <testcase name="userShouldRegisterSuccessFully" time="7.032" classname="com.demo.nopcommerce.testsuit.RegistrationTest" />
      
</testsuite>
